src/docs/testing/index.md

```diff
@@ -84,6 +84,9 @@ such as iOS Simulator or Android Emulator.
 The app under test is typically isolated
 from the test driver code to avoid skewing the results.
 
+See the [integration_test package instructions][] to learn how to write
```

Comment:
I know this might seem like a real nit, but we recommend writing this sentence so that the reader can stop scanning if it doesn't apply. As it is now, you have to read the whole sentence. So:  "For information on how to write integration tests, see...."      https://developers.google.com/style/clause-order?hl=en